src/cljs/main/broadfcui/common/entity_table.cljs

```diff
@@ -153,7 +153,9 @@
          (if (empty? entity-types)
            (on-done {:total-count 0 :filtered-count 0 :results []})
            (let [{:keys [page-number rows-per-page filter-text sort-column sort-order]} query-params
-                 entity-type (nth entity-types selected-filter-index)]
+                 entity-type (if (nil? selected-filter-index)
```

Comment:
Is this the change that avoids the javascript error that was popping up? Looks like it now lists entities of the "first" type if nothing is selected.

This is fine since this is the only place in scope that uses `selected-filter-index`. Another option that might be slightly cleaner would be to set `selected-filter-index` to 0 if it's `nil`. That would encode the "nothing selected means first" logic in a single place that all uses of `selected-filter-index` could benefit from. Since it's only used here, this is definitely a nit than something that needs to be changed.